install.sh

```diff
@@ -2,6 +2,11 @@
 
 set -eu
 
+die() {
+    printf '%s\n' "$1" >&2
+    exit "${2-1}"
```

Comment:
Could you explain this line?

Reply:
Usage is

 ```sh
die MESSAGE [EXIT_STATUS]
```

`>&2` means print to stderr rather than stdout.  `exit ${2-1}` means if `EXIT_STATUS` was not provided, default to `1`.